magento2-nginx/fpm-7.0/etc/confd/templates/magento/env.php.tmpl

```diff
@@ -110,4 +110,15 @@ return array (
         array (
             'date' => 'Tue, 24 Jan 2017 09:12:35 +0000',
         ),
+{{ if getenv "MAGENTO_HTTP_CACHE_HOSTS" }}
+    'http_cache_hosts' => array(
+{{ $varnish := split (replace (replace (getenv "MAGENTO_HTTP_CACHE_HOSTS") "," " " -1) "\n" " " -1) " " }}
```

Comment:
It's based on https://github.com/continuouspipe/dockerfiles/blob/master/nginx/etc/confd/templates/nginx/nginx_ip_whitelist.conf.tmpl#L2 if that helps any :)